Profiles included using --extra-profiles-path are not used by validate

I am loading an extra directory of profiles using the command `--extra-profiles-path` command.

If I list the profiles available I see the added profiles:
```
rocrate-validator -y profiles --extra-profiles-path /Users/user/work/manchester/eScience/HDR-FedA/Cratey-Validator/tests/data/rocrate_va
lidator_profiles list
```
returns:
```
   Available profiles                                                                                                                                                                              
 ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━┓ 
 ┃          Identifier           ┃              URI               ┃ Version ┃              Name              ┃ Description                     ┃        Based on        ┃ Requirements Checks (*) ┃ 
 ┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━┩ 
 │         ro-crate-1.1          │ https://w3id.org/ro/crate/1.1  │   1.1   │       RO-Crate Metadata        │ RO-Crate Metadata               │                        │      REQUIRED: 39       │ 
 │                               │                                │         │       Specification 1.1        │ Specification.                  │                        │     RECOMMENDED: 23     │ 
 │                               │                                │         │                                │                                 │                        │      OPTIONAL: 12       │ 
 │                               │                                │         │                                │                                 │                        │                         │ 
 │        alpha-crate-0.1        │ https://w3id.org/alpha-crate/… │   0.1   │       Alpha RO-Crate 0.1       │ Alpha RO-Crate Metadata         │      ro-crate-1.1      │       REQUIRED: 2       │ 
 │                               │                                │         │                                │ Specification 0.1               │                        │     RECOMMENDED: 0      │ 
 │                               │                                │         │                                │                                 │                        │       OPTIONAL: 0       │ 
 │                               │                                │         │                                │                                 │                        │                         │ 
 │     five-safes-crate-0.4      │ https://w3id.org/5s-crate/0.4  │   0.4   │    Five Safes RO-Crate 0.4     │ Five Safes RO-Crate Metadata    │      ro-crate-1.1      │      REQUIRED: 58       │ 
 │                               │                                │         │                                │ Specification 0.4               │                        │     RECOMMENDED: 54     │ 
 │                               │                                │         │                                │                                 │                        │      OPTIONAL: 17       │ 
 │                               │                                │         │                                │                                 │                        │                         │ 
 │     process-run-crate-0.5     │ https://w3id.org/ro/wfrun/pro… │   0.5   │     Process Run Crate 0.5      │ RO-Crate Metadata               │      ro-crate-1.1      │       REQUIRED: 4       │ 
 │                               │                                │         │                                │ Specification.                  │                        │     RECOMMENDED: 33     │ 
 │                               │                                │         │                                │                                 │                        │      OPTIONAL: 13       │ 
 │                               │                                │         │                                │                                 │                        │                         │ 
 │     workflow-ro-crate-1.0     │ https://w3id.org/workflowhub/… │   1.0   │   Workflow RO-Crate Metadata   │ Workflow RO-Crate Metadata      │      ro-crate-1.1      │       REQUIRED: 8       │ 
 │                               │                                │         │       Specification 1.0        │ Specification.                  │                        │     RECOMMENDED: 7      │ 
 │                               │                                │         │                                │                                 │                        │      OPTIONAL: 12       │ 
 │                               │                                │         │                                │                                 │                        │                         │ 
 │ workflow-testing-ro-crate-0.1 │ https://w3id.org/ro/wftest/0.1 │   0.1   │ Workflow Testing RO-Crate 0.1  │ Workflow Testing RO-Crate       │      ro-crate-1.1      │      REQUIRED: 13       │ 
 │                               │                                │         │                                │ Metadata Specification 0.1      │ workflow-ro-crate-1.0  │     RECOMMENDED: 2      │ 
 │                               │                                │         │                                │                                 │                        │       OPTIONAL: 0       │ 
 │                               │                                │         │                                │                                 │                        │                         │ 
 │    workflow-run-crate-0.5     │ https://w3id.org/ro/wfrun/wor… │   0.5   │     Workflow Run Crate 0.5     │ Workflow Run Crate Metadata     │      ro-crate-1.1      │       REQUIRED: 7       │ 
 │                               │                                │         │                                │ Specification 0.5               │ process-run-crate-0.5  │     RECOMMENDED: 17     │ 
 │                               │                                │         │                                │                                 │ workflow-ro-crate-1.0  │       OPTIONAL: 7       │ 
 │                               │                                │         │                                │                                 │                        │                         │ 
 │   provenance-run-crate-0.5    │ https://w3id.org/ro/wfrun/pro… │   0.5   │    Provenance Run Crate 0.5    │ Provenance Run Crate Metadata   │      ro-crate-1.1      │      REQUIRED: 29       │ 
 │                               │                                │         │                                │ Specification 0.5               │ process-run-crate-0.5  │     RECOMMENDED: 18     │ 
 │                               │                                │         │                                │                                 │ workflow-ro-crate-1.0  │      OPTIONAL: 20       │ 
 │                               │                                │         │                                │                                 │ workflow-run-crate-0.5 │                         │ 
 │                               │                                │         │                                │                                 │                        │                         │ 
 └───────────────────────────────┴────────────────────────────────┴─────────┴────────────────────────────────┴─────────────────────────────────┴────────────────────────┴─────────────────────────┘ 
                                                                           (*) Number of requirements checks by severity                                                                            
                                                                                                                                                                                                    
```

But when I try to use an added profile I get the following error:
```
rocrate-validator -y validate --extra-profiles-path /Users/user/work/manchester/eScience/HDR-FedA/Cratey-Validator/tests/data/rocrate_validator_profiles tests/data/crates/valid/five-safes-crate-result/ -p five-safes-crate
```
returns:
```
  ──────────────────────────────────────────────────────────────────────  ROCrate Validator (ver. 0.8.0_bfbf62f+105-dirty) ───────────────────────────────────────────────────────────────────────  
                                                                                                                                                                                                    


[ERROR] The profile with the identifier "five-safes-crate-0.4" could not be found.
        Please ensure that the profile exists and try again.

        To see the available profiles, run:
        rocrate-validator profiles list
```

--- a/rocrate_validator/models.py
+++ b/rocrate_validator/models.py
@@ -1638,7 +1638,10 @@ class ValidationStatistics(Subscriber):
         # extract the validation settings
         severity_validation = validation_settings.requirement_severity
         profiles: list[Profile] = Profile.load_profiles(
-            validation_settings.profiles_path, severity=severity_validation)
+            validation_settings.profiles_path,
+            extra_profiles_path=validation_settings.extra_profiles_path,
+            severity=severity_validation,
+            allow_requirement_check_override=validation_settings.allow_requirement_check_override)
         profile: Profile = Profile.find_in_list(profiles, validation_settings.profile_identifier)
         target_profile_identifier = profile.identifier
         # initialize the profiles list

--- a/tests/unit/test_cli_internals.py
+++ b/tests/unit/test_cli_internals.py
@@ -81,3 +81,22 @@ def test_compute_stats(fake_profiles_path):
                                _.requirement.profile.identifier == "a"}
 
     logger.error(stats)
+
+
+def test_compute_stats_resolves_profile_from_extra_profiles_path(fake_profiles_path):
+    # ValidationStatistics.__initialise__ used to call Profile.load_profiles
+    # without forwarding extra_profiles_path, so any profile that lived only
+    # under --extra-profiles-path raised ProfileNotFound.
+    settings = ValidationSettings.parse({
+        "profiles_path": DEFAULT_PROFILES_PATH,
+        "extra_profiles_path": fake_profiles_path,
+        "profile_identifier": "a",
+        "enable_profile_inheritance": True,
+        "allow_requirement_check_override": True,
+        "requirement_severity": "REQUIRED",
+    })
+
+    stats = ValidationStatistics.__initialise__(validation_settings=settings)
+
+    assert any(p.identifier == "a" for p in stats["profiles"]), \
+        "Profile 'a' from extra_profiles_path was not resolved by ValidationStatistics"

